builds/misc/images.yaml

```diff
@@ -393,6 +393,22 @@ jobs:
       - powershell: docker system prune -a -f
         displayName: Clean Machine
 
+################################################################################
+  - job: linux_helm_charts
+################################################################################
+    displayName: Helm Charts
+    pool:
+      vmImage: ubuntu-18.04
```

Comment:
Check out the latest changes to images.yaml in `release/1.1`...we're using a 1ES-hosted pool instead of MS-hosted pool for agents. So you'll want to update this `pool` declaration to look like the ones in `release/1.1`. Something like:

```yaml
    pool:
      name: $(pool.linux.name)
      demands:
        - ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker
```